examples/wayland_client.c

```diff
@@ -384,7 +395,12 @@ int main()
 
     wl_output_add_listener(globals->output, &output_listener, NULL);
 
-    while (wl_display_dispatch(display))
+    signal(SIGINT, shutdown);
+    signal(SIGTERM, shutdown);
+    signal(SIGHUP, shutdown);
+    running = 1;
```

Comment:
Nitpick: I expect it's possible for the signal handler to be fired before here, so it'll print Goodbye but still enter the while loop. Perhaps better to set running to true on initiation?

Reply:
It only prints goodbye when running is set

Reply:
Yes you're right. Ignore me